joomla/joomla-cms
# [4.0] Add CSS grid track for debug

Pull Request for Issue #17591 .

### Summary of Changes
Adds a CSS grid track for system debug . Also fixes a minor cascading issue regarding alignment of the component area.

### Testing Instructions

1. Enable system debug in Global Configuration
2. Visit front-end of site

Ensure debug info appears in footer.
